GNUmakefile

```diff
@@ -12,14 +12,14 @@ DOCDIR  ?= $(PREFIX)/share/doc/insomnia
 DATADIR ?= $(PREFIX)/share/insomnia
 
 CFLAGS ?= -Werror -Os
-CFLAGS += -std=c99 -D_POSIX_C_SOURCE=200809L -D_BSD_SOURCE
+CFLAGS += -std=c99 -D_POSIX_C_SOURCE=200809L -D_DEFAULT_SOURCE
 CFLAGS += -Wpedantic -Wall -Wextra
 
 all: $(BINFILES) $(DATFILES) $(LIBFILES)
 bin/%: bin/%.in
-	chmod +w $@
 	sed -e 's|@LIBDIR@|$(LIBDIR)|' \
 		-e 's|@DATADIR@|$(DATADIR)|' < $< > $@
+	chmod +w $@
```

Comment:
The problem now is that the file is no longer rewriteable after the first invocation, hence `sed` can't modify it.

I guess the best fix here would be to just revert a96bc5cb29dba6c2446286f0b7724ae9af19ff45